simtools/simtel/simtel_runner_array.py

```diff
@@ -200,6 +200,7 @@ def has_file(self, file_type, run_number, mode="out"):
 
         info_for_file_name = self.get_info_for_file_name(run_number)
         run_sub_file = self.get_file_name(file_type, **info_for_file_name, mode=mode)
+        self._logger.debug(f"Checking if {run_sub_file} exists")
```

Comment:
as in the other comment, it is not clear where the check is done. I would suggest either to change the message or the name of the function (getting file name and checking if file exists is not the same)